packages/pytest-simcore/src/pytest_simcore/minio_service.py

```diff
@@ -44,7 +44,7 @@ def _ensure_remove_bucket(client: Minio, bucket_name: str):
 @pytest.fixture(scope="module")
 def minio_config(
     docker_stack: Dict, testing_environ_vars: Dict, monkeypatch_module: MonkeyPatch
-) -> Dict[str, str]:
+) -> Dict[str, Any]:

```

Comment:
very minor: python 3.9 allows to use ```dict``` instead of ```Dict```

Reply:
Very nice to know!
@colinRawlings you were wondering about `None` in typing. This is similar.